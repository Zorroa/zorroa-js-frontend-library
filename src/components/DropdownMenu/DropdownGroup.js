import React, { PropTypes, PureComponent } from 'react'
import classnames from 'classnames'

class DropdownGroup extends PureComponent {
  static propTypes = {
    dark: PropTypes.bool.isRequired,
    children: React.PropTypes.any,
  }

  render() {
    const dropdownItemCLasses = classnames('DropdownMenu__group', {
      'DropdownMenu__group--dark': this.props.dark === true,
    })
    return <div className={dropdownItemCLasses}>{this.props.children}</div>
  }
}

export default DropdownGroup
