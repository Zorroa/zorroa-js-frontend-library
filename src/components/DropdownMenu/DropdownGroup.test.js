/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'
import DropdownGroup from './DropdownGroup'

describe('<DropdownGroup />', () => {
  it('It should render the proper markup', () => {
    const tree = renderer.create(<DropdownGroup dark={false} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
