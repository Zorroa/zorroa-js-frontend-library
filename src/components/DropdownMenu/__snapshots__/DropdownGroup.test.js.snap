exports[`<DropdownGroup /> It should render the proper markup 1`] = `
<div
  className="DropdownMenu__group" />
`;
